packages/ui/src/row/row.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import { cn } from "../../helpers";
+import { ERowVariant, rowStyle, TRowVariant } from "./helper";
+
+export interface CustomRowProps extends React.HTMLAttributes<HTMLDivElement> {
+  variant?: TRowVariant;
+  className?: string;
+  children: React.ReactNode;
+}
+
+const CustomRow = React.forwardRef<HTMLDivElement, CustomRowProps>((props, ref) => {
```

Comment:
Call it row, and also can we keep the helpers and types in this file and the component is pretty simple and the information in helpers is key to understand the component.

Reply:
This format was followed in the button component, thats why i have kept it this way.